docs/installation.md

```diff
@@ -6,7 +6,7 @@ Installation
           instance of Treeherder. If you only want to hack on the UI,
           you can just setup a standalone webserver which accesses
           the server backend using node.js, which is much simpler.
-          See the :doc:`UI development section <ui/installation>`.
+          See the [UI development section](ui/installation.md).
```

Comment:
This usage is because this block is in an `eval_rst`, see commit message here:
https://github.com/mozilla/treeherder/commit/eeff0a6fb43722e4d9e01963f508a666437c549d

Reply:
oops, I'll roll that change back.